taxonomies/unused/convert-wikidata-igp.py

```diff
@@ -0,0 +1,61 @@
+import json
+import re
+
+# Load the query.json file
+with open('/Users/pierre/development/openfoodfacts-server/taxonomies/food/query-off-ing.json', 'r') as f:
```

Comment:
Did you mean to hardcode this (and other) paths? Would it be possible to use a path relatively to the script (`../food/query-off-ing.json`)? I’m pretty sure there’s a Python function/method to get location of current script/file which might be useful for this, but I don’t remember what it is offhand.